tests/models/test_dagbag.py

```diff
@@ -91,6 +91,15 @@ def test_get_non_existing_dag(self):
         non_existing_dag_id = "non_existing_dag_id"
         assert dagbag.get_dag(non_existing_dag_id) is None
 
+    def test_serialized_dag_not_existing_doesnt_raise(self):
+        """
+        test that retrieving a non existing dag id returns None without crashing
+        """
+        dagbag = models.DagBag(dag_folder=self.empty_dir, include_examples=False, read_dags_from_db=True)
+
+        non_existing_dag_id = "non_existing_dag_id"
+        assert dagbag.get_dag(non_existing_dag_id) is None
+
```

Comment:
Does this test really check the code path you think it does? i.e. without your change does this throw the (now deleted) SerializedDagNotFound error?

Reply:
Yes. It throws the SerializedDagNotFound error without my change